rfcs/0000-template.md

```diff
@@ -19,6 +19,12 @@ Please focus on explaining the motivation so that if this RFC is not accepted, t
 
 This is the bulk of the RFC. Explain the component and the design in enough detail for someone familiar with design systems and web components can understand, and someone familiar with the implementation help to implement it. This should get into as many specifics as possible, outline any edge cases or extra consideration, and include examples of how it can be used. Any new terminology should also be defined here.
 
+It's important to ensure your proposal address the key goals of the OpenUI Design System to create a component that is:
```

Comment:
this sentence became a bit wooly, and besides it's not the goal to create a component but to deliver components that help with these aspects.

What about:

```suggestion
Ensure that your proposal shows how your component aids authors in building products that are:
```